Tests/PowerSkillTests/GeoPointFromNameTests/GeoPointFromNameTests.cs

```diff
@@ -20,17 +21,22 @@ public static void Setup(TestContext context)
             WebApiSkillHelpers.TestMode = true;
             WebApiSkillHelpers.TestWww = req =>
             {
-                string query = req.RequestUri.ParseQueryString()["query"];
+                string query = req.RequestUri.ParseQueryString()["q"];
```

Comment:
FYI, there was a test broken because of the recent API contract change not being applied here, so I just went ahead and fixed it, all tests are passing now :)

Reply:
Thanks!